### PR TITLE
Cordoba - fixed image link

### PR DESCRIPTION
**Pull request by uclWerner**

### CODE REVIEW

File: web_source/cities/south_america/argentina/cordoba.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ koɾðoβa
 ### What we would like to see there
 
 If you're into religion, its got a really lovely church
-![image of a church](https://commons.wikimedia.org/wiki/File:Catedral_de_C%C3%B3rdoba,_Argentina_3.jpg)
+![image](https://commons.wikimedia.org/wiki/File:Catedral_de_C%C3%B3rdoba,_Argentina_3.jpg)
```
Comment (jonc125) [inline web_source/cities/south_america/argentina/cordoba.md]: This link is to the HTML page with info about the image, not the image itself, which is why it doesn't work. Descriptive alt-text is also good ;)
```suggestion
![Photo of a cathedral](https://upload.wikimedia.org/wikipedia/commons/thumb/3/36/Catedral_de_C%C3%B3rdoba%2C_Argentina_3.jpg/64px-Catedral_de_C%C3%B3rdoba%2C_Argentina_3.jpg?20060610023555)```

**Review by jonc125 — CHANGES_REQUESTED** [review]

File: web_source/cities/south_america/argentina/cordoba.md [hunk 2]
```diff
@@ -9,7 +9,7 @@ koɾðoβa
...
 
 ### Extra information
```
Comment (TanyaRoum) [inline web_source/cities/south_america/argentina/cordoba.md:14]: Extra information can be added.

File: web_source/cities/south_america/argentina/cordoba.md [hunk 3]
```diff
@@ -9,7 +9,7 @@ koɾðoβa
 ### What we would like to see there
```
Comment (TanyaRoum) [inline web_source/cities/south_america/argentina/cordoba.md:9]: looks nice

**Review by TanyaRoum — CHANGES_REQUESTED** [review]

**Review by arc-teaching — APPROVED** [review]
Closes #54

**Pull request merged**